site/source/tutorials/10XX_development_environment/index.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: tutorial
+title: Setting up a development environment
+overview: This tutorial shows how to set up and use a development environment that takes advantage of Dojo 2 features.
+---
+
+{% section 'first' %}
+
+# Development Environment
+
+## Overview
+
+This tutorial will cover how to set up your tools and environment for the best development experience. You can use demo code from any of the tutorials, but this tutorial will reference finished code from the [Forms tutorial](../005_form_widgets/).
+
+## Prerequisites
+You can [download](../assets/10xx_development_environment-initial.zip) a fresh install of the demo code to get started.
+
+You should be familiar with the developer tools in your browser of choice. This tutorial will reference the Chrome developer tools, but the instructions should apply to any browser.
+
+{% section %}
+
+## Command line tools
+
+The [first tutorial](../000_local_installation/) goes into installing the Dojo CLI, and the available commands. Before that point, we assume you have an environment with supported versions of node, npm, and git installed.
+
+While git is not strictly necessary to use Dojo 2, a robust development environment also needs some sort of version control system. All of Dojo 2 uses git as the version control tool and we manage all of our code on GitHub. Several of READMEs and other instructions will assume that git is being used for version control. If it or another version control tool is not already installed, you can follow [these instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
```

Comment:
If it or another version control tool is not already installed, you can follow [these instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). -> If `git` or another version control tool is not already installed, you can follow [the Git installation instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).